### ISSUE TITLE
[REQUEST]: New Observability threshold rule

### ISSUE THREAD

**Issue by lcawl** [elastic/observability-docs#3031]
### Description

Per https://github.com/elastic/actionable-observability/issues/36 there's a new rule that will combine three previous (logs threshold, infrastructure threshold, and metrics threshold) Observability rules, though I think all four will continue to exist in parallel for a little while.

Per @katrin-freihofner she has discussed with @mdbirnstiehl and they have added a link from the new rule to the docs (currently just a stub file), but will ultimately be similar to the existing rule-specific pages in this section: https://www.elastic.co/guide/en/observability/master/create-alerts.html

### Resources

- https://github.com/elastic/actionable-observability/issues/36
- Design [figma file](https://www.figma.com/file/WtGj2hHUPVGTpQjSs1TtTe/Rule-creation-experience?type=design&mode=design&t=PlaI1ITWRFar8FfZ-0)
- https://github.com/elastic/kibana/issues/160045

### Collaboration

TBD. The docs and product team will work together to determine the best path forward.

### Point of contact.

**Main contact:**

**Stakeholders:**
@katrin-freihofner

**Comment by katrin-freihofner** [comment]
This rule is behind a feature flag - we can postpone the documentation for it to 8.10

**Comment by mdbirnstiehl** [comment]
Due to a privilege issue, this has been pushed to 8.11.
